draft-smyslov-ipsecme-rfc8229bis.xml

```diff
@@ -1102,15 +1102,11 @@
   2)  ---------------------  TLS Session  ---------------------
       ClientHello               ---------->
                                                     ServerHello
-                                                   Certificate*
-                                             ServerKeyExchange*
-                                <----------     ServerHelloDone
-      ClientKeyExchange
-      CertificateVerify*
-      [ChangeCipherSpec]
-      Finished                  ---------->
-                                             [ChangeCipherSpec]
-                                <----------            Finished
+                                          {EncryptedExtensions}
```

Comment:
I'm not very familiar with TLS 1.3 :-), but should't initial Client's message also include key_share, signature_algorithms, psk_key_exchange_modes and pre_shared_key? All of them are optional, but some of them must be present, as far as I understand from RFC8446.